v2/azure-arm.yaml

```diff
@@ -56,12 +56,12 @@ typeFilters:
     because: Some types (SitesSlotsConfig) use OneOf in a way we don't currently handle. Exclude the whole set for now.
   - action: prune
     group: network
-    version: 'v*api201*'
-    because: Pruning all older networking api versions types since we don't use them anyway
+    version: 'v*beta201*'
```

Comment:
I included `beta` here because the wildcard was too general - matching just `v*201*` would pick up other API versions as well - e.g. we'd prune any February version, `v20220201`.

Also, these type filters are applied before we do the cloning to create the `alpha` variants, so they get pruned too.

Reply:
Changed to explicitly list the years that need exclusion.